core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java

```diff
@@ -93,7 +93,7 @@ public Transaction newCreateTableTransaction(
     String baseLocation = location != null ? location : defaultWarehouseLocation(identifier);
     Map<String, String> tableProperties = properties != null ? properties : Maps.newHashMap();
     TableMetadata metadata = TableMetadata.newTableMetadata(schema, spec, baseLocation, tableProperties);
-    return Transactions.createTableTransaction(ops, metadata);
+    return Transactions.createTableTransaction(identifier.name(), ops, metadata);
```

Comment:
Did you mean to use `identifier.toString` here as well?

Reply:
Fixed.